### PR TITLE
Add general documentation and ripple.md.

### PR DESCRIPTION
**Pull request by PJarali-NI**
# Added Readme.md content
# Added images folder under docs
# Added Ripple.md

### CODE REVIEW

File: README.md
Comment (ngoliNI) [inline README.md:1]: Add a getting started section with help me link; add contributing section as well.

File: docs/sw-dev.md
Comment (ngoliNI) [inline docs/sw-dev.md:1]: the link for VIPM points towards measurement link LabVIEW not python, also update the software dependencies and VIPM packages sections accordingly.
remove * at the end.

File: docs/measurements/ripple.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## InstrumentStudio Panel
+
+### Usage
+
+1. Select the appropriate source, load and scope resource names and update other parameters as needed. Please note that, the measurement is in 'Perform Measurement' mode of operation by default.
+
+   ![alt text](meas-images/ripple-config.png)
+
+2. Run the measurement. The Ripple Voltage values are displayed on the graph. The RMS and Peak-to-Peak values of ripple are calculated and displayed in the table along with voltage and current values of source and load devices.
```
Comment (ngoliNI) [inline docs/measurements/ripple.md:16]: please remove table as in python we don't have it.

**Review by ngoliNI — CHANGES_REQUESTED** [review]

File: docs/sw-dev.md
Comment (PJarali-NI) [reply docs/sw-dev.md:1]: Removed VIPMs and * as they are not required.

File: README.md
Comment (PJarali-NI) [reply README.md:1]: Added contribution section.

File: docs/measurements/ripple.md (hunk 1 above)
Comment (PJarali-NI) [reply docs/measurements/ripple.md]: removed table and made it as indicators.

**Review by ngoliNI — APPROVED** [review]

**Pull request merged**